[2024-08-26T18:34:03.700+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-26T18:34:03.700+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-26T18:34:03.724+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-26T18:34:03.725+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-26T18:34:03.745+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-26T18:34:03.754+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-26T17:00:00+00:00 [queued]>
[2024-08-26T18:34:03.762+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-26T17:00:00+00:00 [queued]>
[2024-08-26T18:34:03.762+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-08-26T18:34:03.774+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): calculate_exchange_rate_change> on 2024-08-26 17:00:00+00:00
[2024-08-26T18:34:03.778+0000] {standard_task_runner.py:57} INFO - Started process 221 to run task
[2024-08-26T18:34:03.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'currency_exchange_dag', 'calculate_exchange_rate_change', 'scheduled__2024-08-26T17:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rate_dag.py', '--cfg-path', '/tmp/tmp59qq_rnb']
[2024-08-26T18:34:03.784+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask calculate_exchange_rate_change
[2024-08-26T18:34:03.795+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-26T18:34:03.827+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509
[2024-08-26T18:34:03.827+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=currency_exchange_dag/run_id=scheduled__2024-08-26T17:00:00+00:00/task_id=calculate_exchange_rate_change'
[2024-08-26T18:34:03.829+0000] {task_command.py:410} INFO - Running <TaskInstance: currency_exchange_dag.calculate_exchange_rate_change scheduled__2024-08-26T17:00:00+00:00 [running]> on host 5f8e3ab10fa1
[2024-08-26T18:34:03.839+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-08-26T18:34:03.908+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_exchange_dag' AIRFLOW_CTX_TASK_ID='calculate_exchange_rate_change' AIRFLOW_CTX_EXECUTION_DATE='2024-08-26T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-26T17:00:00+00:00'
[2024-08-26T18:34:03.915+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-08-26T18:34:03.924+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=currency_exchange_dag, task_id=calculate_exchange_rate_change, execution_date=20240826T170000, start_date=20240826T183403, end_date=20240826T183403
[2024-08-26T18:34:03.954+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-08-26T18:34:03.967+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
